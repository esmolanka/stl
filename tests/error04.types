
type Foo (a : Type) = a a
