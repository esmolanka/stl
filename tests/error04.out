tests/error04.stl:2:23-2:24: error:
    Kind mismatch while checking type:
        a
    Expected kind:
        Type
    Actual kind:
        ?κ₁ -> κ₂